bin/metrics-docker-stats.rb

```diff
@@ -212,4 +220,17 @@ def blkio_stats(io_stats)
     end
     stats_out
   end
+
+  def calculate_cpu_percent(stats)
+    cpu_percent = 0.0
+    previous_cpu = stats['precpu_stats']['cpu_usage']['total_usage']
+    previous_system = stats['precpu_stats']['system_cpu_usage']
+    cpu_delta = stats['cpu_stats']['cpu_usage']['total_usage'] - previous_cpu
+    system_delta = stats['cpu_stats']['system_cpu_usage'] - previous_system
+    if system_delta > 0 && cpu_delta > 0
+      number_of_cpu = stats['cpu_stats']['cpu_usage']['percpu_usage'].length
+      cpu_percent = (cpu_delta.to_f / system_delta.to_f) * number_of_cpu * 100
+    end
+    cpu_percent
```

Comment:
Reduce the number of decimal points going back?
Using sprintf with the closest percentage:
```
sprintf('%.2f', cpu_percent)
```
Using round for the 1/100th of a percentage:
```
cpu_percent.round(4)
```

Reply:
hi there, sorry for the late reply. I agree than not limiting the number of decimals here it's a mistake. I think with 2 decimal digits it's more than enough. I'm pushing the changes you proposed. Thanks!